external/js/cothority/src/byzcoin/byzcoin-rpc.ts

```diff
@@ -102,24 +107,28 @@ export default class ByzCoinRPC implements ICounterUpdater {
 
         const ret = await rpc.conn.send<CreateGenesisBlockResponse>(req, CreateGenesisBlockResponse);
         rpc.genesis = ret.skipblock;
-        rpc.latest = ret.skipblock;
+        rpc._latest = ret.skipblock;
         await rpc.updateConfig();
 
         return rpc;
     }
-
+    private _latest: SkipBlock;
     private genesisDarc: Darc;
     private config: ChainConfig;
     private genesis: SkipBlock;
-    private latest: SkipBlock;
     private conn: IConnection;
 
-    protected constructor() {}
+    protected constructor() {
+    }
 
     get genesisID(): InstanceID {
         return this.genesis.computeHash();
     }
 
+    get latest(): SkipBlock {
+        return new SkipBlock(this._latest);
```

Comment:
It a immutable (js style) object, that should be enough.
```suggestion
        return this._latest;
```

Reply:
Oh my - I hate getters and setters that don't do anything. 

(deleted 20 lines of rant)